backend-db/modules/db/couchdb/index.js

```diff
@@ -402,12 +403,23 @@ const layer = {
         create.hashtags = create.hashtags || [];
 
         const id = util.uuid.random();
+        const account = await layer.getAccount(who);
         await adapter.createDocument('Trashpoint', id, create, {
             updatedAt: util.time.getNowUTC(),
             updatedBy: who,
             createdAt: util.time.getNowUTC(),
             createdBy: who,
         });
+        if(account && account.team) {
+            await adapter.createDocument('TeamTrashpoint', id, {
+                team: account.team
+            }, {
+                updatedAt: util.time.getNowUTC(),
+                updatedBy: who,
+                createdAt: util.time.getNowUTC(),
+                createdBy: who,
+            });
+        }
```

Comment:
It's unclear why there needs to be a separate document collection TeamTrashpoint which holds only the team that created a trashpoint. Why can't this information be added directly to the trashpoints collection?